.gitignore

```diff
@@ -202,3 +202,4 @@ shellcheck*/
 
 # Ingore moe/marlin_moe gen code
 csrc/moe/marlin_moe_wna16/kernel_*
+uv.lock
```

Comment:
quick dropby, I think we should move to uv.lock at some point, so we shouldn't ignore this lock file here.